<!DOCTYPE html>
<html>
  <head>
      <title>Science</title>
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link href="css/textrender.css" rel="stylesheet">
      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
      <script src="script.js"></script>
  </head>
  <body>
    <main class="paper">
      <button id='prev' class="paper__page-prev">prev page ></button>
      <button id='next' class="paper__page-next">next page ></button>
      <header class="paper__header">
        <ul class="paper__identifiers">      
          <li class="paper__identifiers-item">
            Open Access
          </li>
          <li class="paper__identifiers-item">
            Published: <time datetime="2011-01-26">26 January 2011</time>
          </li>
          <li class="paper__identifiers-item">
            Source: <a href="https://www.nature.com/articles/nature09687">https://www.nature.com/articles/nature09687</a>
          </li>
        </ul>
        <h1 class="paper__title">
          Comparative and demographic analysis of orang-utan genomes
        </h1>
        <ul class="paper__authors">
          <li class="paper__authors__item">
            <a href="#0">Devin P. Locke</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">LaDeana W. Hillier</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Wesley C. Warren</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Kim C. Worley</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Lynne V. Nazareth</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Donna M. Muzny</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Shiaw-Pyng Yang</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Zhengyuan Wang</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Asif T. Chinwalla</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Pat Minx</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Makedonka Mitreva</a>
          </li>
          <li class="paper__authors__item">
            <a href="#0">Lisa Cook</a>
          </li>
        </ul>
        <p class="paper__journal">                             
          <a href="#0"><i data-test="journal-title">Nature</i></a>
          <strong>volume 469</strong>, pages 529 - 533 2011
        </p>
        <ul class="paper__metrics">
          <li class="paper__metrics-item">
            <span class="paper__metrics-item-number">3766</span> Accesses
          </li>
          <li class="paper__metrics-item">
            <span class="paper__metrics-item-number">329</span> Citations
          </li>
          <li class="paper__metrics-item">
            <span class="paper__metrics-item-number">136</span> Altmetric
          </li>
        </ul>
      </header>
      <article class="paper__text">
        <section class="paper__abstract">
          <h2>Abstract</h2>
          <p>
            "Orang-utan" is derived from a Malay term meaning "man of the forest" and aptly describes the southeast Asian great apes native to Sumatra and Borneo. The orang-utan species, <i>Pongo abelii</i> (Sumatran) and <i>Pongo pygmaeus</i> (Bornean), are the most phylogenetically distant great apes from humans, thereby providing an informative perspective on hominid evolution. Here we present a Sumatran orang-utan draft genome assembly and short read sequence data from five Sumatran and five Bornean orang-utan genomes. Our analyses reveal that, compared to other primates, the orang-utan genome has many unique features. Structural evolution of the orang-utan genome has proceeded much more slowly than other great apes, evidenced by fewer rearrangements, less segmental duplication, a lower rate of gene family turnover and surprisingly quiescent Alu repeats, which have played a major role in restructuring other primate genomes. We also describe a primate polymorphic neocentromere, found in both <i>Pongo</i> species, emphasizing the gradual evolution of orang-utan genome structure. Orang-utans have extremely low energy usage for a eutherian mammal<sup><a class="paper__ref-link" href="#ref-CR1">1</a></sup>, far lower than their hominid relatives. Adding their genome to the repertoire of sequenced primates illuminates new signals of positive selection in several pathways including glycolipid metabolism. From the population perspective, both <i>Pongo</i> species are deeply diverse; however, Sumatran individuals possess greater diversity than their Bornean counterparts, and more species-specific variation. Our estimate of Bornean/Sumatran speciation time, 400,000 years ago, is more recent than most previous studies and underscores the complexity of the orang-utan speciation process. Despite a smaller modern census population size, the Sumatran effective population size (<i>N</i><sub>e</sub>) expanded exponentially relative to the ancestral <i>N</i><sub>e</sub> after the split, while Bornean <i>N</i><sub>e</sub> declined over the same period. Overall, the resources and analyses presented here offer new opportunities in evolutionary genomics, insights into hominid biology, and an extensive database of variation for conservation efforts.
          </p>
          <a href="#0" class="paper__download">
              Download PDF
          </a>
        </section>
        <section class="paper__body">
          <h2>Main</h2>
          <p>Orang-utans are the only primarily arboreal great apes, characterized by strong sexual dimorphism and delayed development of mature male features, a long lifespan (35–45 years in the wild, more than 55 years in captivity) and the longest interbirth interval among mammals (8 years on average)<sup><a class="paper__ref-link" href="#ref-CR2">2</a></sup>. Orang-utans create and adeptly use tools in the wild, and while long presumed socially solitary, dense populations of Sumatran orang-utans show complex social structure and geographic variability in tool use indicative of cultural learning<sup><a class="paper__ref-link" href="#ref-CR3">3</a></sup>. Both species have been subject to intense population pressure from loss of habitat, deforestation, hunting and disease. A 2004 study estimated that 7,000–7,500 Sumatran individuals and 40,000–50,000 Bornean individuals remained in the wild in fragmented subpopulations<sup><a class="paper__ref-link" href="#ref-CR4">4</a>,<a class="paper__ref-link" href="#ref-CR5">5</a></sup>. The International Union for Conservation of Nature lists Sumatran orang-utans as critically endangered and Bornean orang-utans as endangered.</p><p>We sequenced the genome of a female Sumatran orang-utan using a whole-genome shotgun strategy. The assembly provides 5.5-fold coverage on average across 3.08 gigabases (Gb) of ordered and oriented sequence (<a class="paper__ref-link" href="#Tab1">Table 1</a>) (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 1). Accuracy was assessed by several metrics, including comparison to 17 megabases (Mb) of finished bacterial artificial chromosome (BAC) sequences and a novel method of detecting spurious insertions and deletions (<a class="paper__ref-link" href="#ref-MOESM279">Supplementary Information</a> section 2). Further validation resulted from orang-utan–human divergence estimates based on alignment of whole-genome shotgun reads to the human reference (Hs.35; <a  class="paper__ref-link" href="#Fig1">Fig. 1</a>, <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 3). We also sequenced the genomes of 10 additional unrelated wild-caught orang-utans, five Sumatran and five Bornean, using a short read sequencing platform (297 Gb of data in total; <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 4). The orang-utan gene set was constructed using a combination of human gene models and orang-utan complementary DNA data generated for this project (available at <a href="http://www.ensembl.org/Pongo_pygmaeus/Info/StatsTable">http://www.ensembl.org/Pongo_pygmaeus/Info/StatsTable</a>; see also <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 5).</p>
          <figure class="paper__figure">
            <figcaption id="Tab1">Table 1: Sumatran orang-utan assembly statistics</figcaption>
            <a class="paper__link-ext" href="https://www.nature.com/articles/nature09687/tables/1">
              View table
            </a>
          </figure>
          <figure class="paper__figure">
            <figcaption id="Fig1">
              Figure 1: Divergence among great apes, a small ape, and an Old World monkey with respect to humans.
            </figcaption>
            <img src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_Article_BFnature09687_Fig1_HTML.jpg" alt="figure1" loading="lazy">
            <p>
              We estimated nucleotide divergence in unique gap-free sequence, indicated at each node, from the alignment of rhesus macaque (yellow), gibbon (purple), orang-utan (orange), gorilla (aqua), chimpanzee (green) and human (blue) whole genome shotgun reads to the human reference (Hs.35; <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 3). Note that the Bornean (<i>P. pygmaeus</i>) and Sumatran (<i>P. abelii</i>) orang-utan species showed nucleotide identity comparable to that of bonobo (<i>Pan paniscus</i>) and chimpanzee (<i>Pan troglodytes</i>). Estimates of divergence time based on sequence identity are indicated at each node, ∼1 Myr implies approximately 1 Myr or less. Values taken from refs <a class="paper__ref-link" href="#ref-CR29">29</a> and <a class="paper__ref-link" href="#ref-CR30">30</a> where indicated.
            </p>
            <p>
              <a class="paper__ext-link" href="#MOESM274">PowerPoint slide</a>
            </p>
          </figure>
          <p>
            Among hominids, the orang-utan karyotype is the most ancestral<sup><a class="paper__ref-link" href="#ref-CR6">6</a></sup>, and sequencing the orang-utan genome allowed a comprehensive assessment of conservation among the wide range of rearrangement types and sequence classes involved in structural variation. We characterized orang-utan synteny breaks in detail cytogenetically in concert with an <i>in silico</i> approach that precisely tracked rearrangements between primate (human, chimpanzee, orang-utan and rhesus macaque) and other mammalian assemblies (mouse, rat and dog) (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 6). Alignment-level analyses at 100 kilobase (kb) and 5 kb resolution found that the orang-utan genome underwent fewer rearrangements than the chimpanzee or human genomes, with a bias for large-scale events (&gt;100 kb) on the chimpanzee branch (<a class="paper__ref-link" href="#Tab2">Table 2</a>). Orang-utan large-scale rearrangements were further enriched for segmental duplications (52%) than for small-scale events (27%), suggesting that mechanisms other than non-allelic homologous recombination may have made a greater contribution to small rearrangements. Genome-wide, we estimated less segmental duplication content (3.8% total) in the orang-utan genome compared to the chimpanzee and human genomes (5%) using equivalent methods (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 11). We also assessed the rate of turnover within gene families as an additional measure of genome restructuring (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 12). Our analysis indicated that the human and chimpanzee lineages, as well as their shared ancestral lineage after the orang-utan split, had the highest rates of gene turnover among great apes (0.0058 events per gene per Myr)—more than twice the rate of the orang-utan and macaque lineages (0.0027)—even as the nucleotide substitution rate decreased<sup><a class="paper__ref-link" href="#ref-CR7">7</a></sup>. Collectively, these data strongly suggest that structural evolution proceeded much more slowly along the orang-utan branch, in sharp contrast to the acceleration of structural variation noted for the chimpanzee and human genomes<sup><a class="paper__ref-link" href="#ref-CR8">8</a>,<a class="paper__ref-link" href="#ref-CR9">9</a></sup>.
          </p>
          <figure class="paper__figure">
            <figcaption id="Tab2">
              Table 2 Number of genome rearrangements by species
            </figcaption>
             <a class="paper__link-ext" href="/articles/nature09687/tables/2">
              View table
            </a>
          </figure>
          <p>
            One structural variant that we characterized in detail was a previously described polymorphic "pericentric inversion" of orang-utan chromosome 12 (ref. <a class="paper__ref-link" href="#ref-CR10">10</a>). Surprisingly, both forms of this chromosome showed no difference in marker order by fluorescence <i>in situ</i> hybridization (FISH) despite two distinct centromere positions—the hallmark of a neocentromere (<a class="paper__ref-link" href="#Fig2">Fig. 2</a>; <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 8). Neocentromere function was confirmed by chromatin immunoprecipitation with antibodies to centromeric proteins CENP-A and CENP-C and subsequent oligo array hybridization (ChIP-on-chip), which narrowed the neocentromere to a ~225 kb gene-free window devoid of α satellite-related sequences. Our observations bore similarity to a recently described centromere repositioning event in the horse genome<sup><a class="paper__ref-link" href="#ref-CR11">11</a></sup>; however, this is to our knowledge the first observation of such a variant among primates, with the additional complexity of polymorphism in two closely related species. Potentially related, orang-utan chromosome 12 did not show any appreciable centromeric alphoid FISH signal in comparison to other autosomes. The neocentromere most probably arose before the Bornean/Sumatran split as it is found in both species, and represents a unique opportunity to study the initial stages of centromere formation and the impact of such a large chromosomal variant on population variation and recombination.
          </p>
          <figure class="paper__figure">
            <figcaption id="Fig2">
              Figure 2: The neocentromere of orang-utan chromosome 12.
            </figcaption>
            <img src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_Article_BFnature09687_Fig2_HTML.jpg" alt="figure2" loading="lazy">
            <p>
              Note the identical order of four sequentially arranged BAC-derived FISH probes indicated in yellow, blue, green and red (given in <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 8) between the normal (<b>a</b>) and neocentromere-bearing (<b>b</b>) configurations of orang-utan chromosome 12, despite discordant centromere positions. The left image of <b>a</b> and the right image of <b>b</b> are DAPI-only images that show the primary constriction of both chromosomal forms, indicated by the arrows. The neocentromere recruits centromeric proteins CENP-A and CENP-C and lies within a ~225 kb gene-free and α satellite-free region. The neocentromere-bearing variant is polymorphic in both Bornean and Sumatran populations, suggesting the neocentromere arose before the Bornean/Sumatran split, yet has not been fixed in either species.
            </p>
            <p>
              <a class="paper__ext-link" href="#MOESM275">PowerPoint slide</a>
            </p>
          </figure>
          <p>
            The orang-utan genome has a comparable cadre of mobile elements to that of other primates, comprising roughly half the genome<sup><a class="paper__ref-link" href="#ref-CR12">12</a>,<a class="paper__ref-link" href="#ref-CR13">13</a>,<a class="paper__ref-link" href="#ref-CR14">14</a></sup>. Orang-utan long interspersed element 1 (LINE1; L1) and SVA (SINE-R, VNTR and Alu) element expansions were expectedly broad, with roughly 5,000 and 1,800 new insertions respectively, consistent with other primates (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 9). Surprisingly, Alu elements were relatively quiescent, with only ~250 recent insertions identified by computational and laboratory approaches (<a class="paper__ref-link" href="#Fig3">Fig. 3</a>). By comparison, 5,000 human-specific and 2,300 chimpanzee-specific Alu elements were identified by similar methods. The rate of processed pseudogene formation, which like Alu insertion requires functional L1 machinery<sup><a class="paper__ref-link" href="#ref-CR15">15</a></sup>, was similar for the human (8.0 per Myr), chimpanzee (12.7 per Myr) and orang-utan (11.6 per Myr) lineages (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 10). We identified a small number of polymorphic Alu elements exclusive to <i>P. abelii</i> (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 19), indicating that Alu retroposition has been strongly limited, but not eliminated. This dramatic Alu-specific repression represents an unprecedented change in primate retrotransposition rates<sup><a class="paper__ref-link" href="#ref-CR16">16</a>,<a class="paper__ref-link" href="#ref-CR17">17</a></sup>. Possible explanations include L1 source mutations that lowered Alu affinity and <i>cis</i> mobilization preference<sup><a class="paper__ref-link" href="#ref-CR18">18</a></sup>, pressure against Alu retroposition from the <i>APOBEC</i> RNA editing family<sup><a class="paper__ref-link" href="#ref-CR19">19</a></sup>, or fixation of less effectively propagated Alu "master" variants.
          </p>
          <figure class="paper__figure">
            <figcaption id="Fig3">
              Figure 3: Alu quiescence in the orang-utan lineage.
            </figcaption>
            <img src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_Article_BFnature09687_Fig3_HTML.jpg" alt="figure2" loading="lazy">
            <p>
              We identified only ~250 lineage-specific Alu retroposition events in the orang-utan genome, a dramatically lower number than that of other sequenced primates, including humans. The total number of lineage-specific L1, SVA and Alu insertions is shown (pie chart) at the terminus of each branch of the phylogeny of sequenced great apes shown in grey at left, along with the rate of insertion events per element type (bar graph). Reduced Alu retroposition potentially limited the effect of a wide variety of repeat-driven mutational mechanisms in the orang-utan lineage that played a major role in restructuring other primate genomes.
            </p>
            <p>
              <a class="paper__ext-link" href="#MOESM276">PowerPoint slide</a>
            </p>
          </figure>
          <p>
            It is tempting to propose a correlation between reduced Alu retroposition and the greater structural stability of the orang-utan genome. More than 10<sup>6</sup> Alu elements exist within primate genomes. Because of their large copy number and high sequence identity, Alu repeats play a crucial role in multiple forms of structural variation through insertion and post-insertion recombination<sup><a class="paper__ref-link" href="#ref-CR20">20</a></sup>. By virtue of reduced Alu retroposition, the orang-utan lineage experienced fewer new insertions and a putative decrease in the number of regions susceptible to post-insertion Alu-mediated recombination events genome-wide, limiting the overall mobile element threat to the genome.
          </p>
          <p>
            The unique phylogenetic position of <i>Pongo</i> species also offered the opportunity to detect signals of positive selection with increased power. We assessed positive selection in 13,872 human genes with high-confidence orthologues in the orang-utan genome, and in one or more of the chimpanzee, rhesus macaque and dog genomes, using branch-site likelihood ratio tests (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 15)<sup><a class="paper__ref-link" href="#ref-CR14">14</a>,<a class="paper__ref-link" href="#ref-CR21">21</a></sup>. Two new Gene Ontology categories were statistically enriched for positive selection in primates: "visual perception" and "glycolipid metabolic processes"<sup><a class="paper__ref-link" href="#ref-CR22">22</a></sup>. The enrichment for visual perception includes strong evidence from two major visual signalling proteins: arrestin (<i>SAG</i>, <i>P</i> = 0.007) and recoverin (<i>RCVRN</i>, <i>P</i> = 0.008), as well as the opsin, <i>OPN1SW1</i> (<i>P</i> = 0.020), associated with blue colour vision<sup><a class="paper__ref-link" href="#ref-CR23">23</a></sup>. The enrichment for glycolipid metabolism is particularly intriguing owing to medium-to-strong evidence for positive selection (nominal <i>P</i> &lt; 0.05) from six genes expressed in nervous tissue that cluster in the cerebroside-sulphatid region of the sphingolipid metabolism pathway (<a class="paper__ref-link" href="#Fig4">Fig. 4</a>). This pathway is associated with human neurodegenerative diseases such as Gaucher"s, Sandhoff"s, Tay-Sachs, and metachromatic leukodystrophy. Variation in lipid metabolism may have affected neurological evolution among primates, and diversity of diets and life history strategies, as apes—especially orang-utans—have slower rates of reproduction and dramatically lower energy usage than other primates and mammals<sup><a class="paper__ref-link" href="#ref-CR1">1</a></sup>.
          </p>
          <figure class="paper__figure">
            <figcaption id="Fig4">
              Figure 4: Enrichment for positive selection in the cerebroside-sulphatid metabolism pathway.
            </figcaption>
            <img src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_Article_BFnature09687_Fig4_HTML.jpg">
            <p>
              We identified six genes (indicated in yellow) under moderate to strong positive selection in primates (<i>P</i> < 0.05) that fall within the cerebroside-sulphatid region of the sphingolipid metabolism pathway (adapted from human KEGG pathway 00600). This pathway is associated with several human lysosomal storage disorders, such as Gaucher"s disease, Sandhoff"s disease, Tay-Sachs disease and metachromatic leukodystrophy. Abbreviations, annotations and connections are presented in accordance with KEGG standards: solid lines represent direct relationships between enzymes (boxes) and metabolites (circular nodes), dashed lines represent indirect relationships, arrowheads denote directionality (see <a href="http://www.genome.jp/kegg-bin/show_pathway?map00600">http://www.genome.jp/kegg-bin/show_pathway?map00600</a> for further details).
            </p>
            <p>
              <a class="paper__ext-link" href="#MOESM277">PowerPoint slide</a>
            </p>
          </figure>
          <p>
            Ancestral orang-utan species ranged broadly across southeast Asia, including the mainland, while modern species are geographically restricted to their respective islands owing to environmental forces and human population expansion. Historically, protein markers, restriction fragment length polymorphisms, and small sets of mitochondrial and nuclear markers have been used to estimate the divergence and diversity of orang-utan species. We used short read sequencing to address this question from a genome-wide perspective. We first estimated average Bornean/Sumatran nucleotide identity genome-wide (99.68%) based on the alignment of 20-fold coverage of short read data from a Bornean individual to the Sumatran reference (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 16). We then called single nucleotide polymorphisms (SNPs) from the alignment of all short read data from 10 individuals (five Bornean, including the 20-fold coverage mentioned above, and five Sumatran) (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 4). We analysed each species separately using a Bayesian approach with 92% power to detect SNPs (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 20). Because of relatively deep sequencing, allele frequency spectra were estimated accurately, but with an overestimation of singletons compared to other allele frequency categories of approximately 7.8% based on re-sequencing a subset of SNPs (<i>n</i> = 108) (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 20). This level of error had only a marginal effect on downstream population genetic analyses (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 21). Overall, 99.0% (931/940) of genotypes were accurately called within the re-sequenced subset of SNPs.</p><p>In total, we identified 13.2 × 10<sup>6</sup> putative SNPs across 1.96 Gb of the genome, or 1 SNP every 149 base pair (bp) on average. Within the Bornean and Sumatran groups we detected 6.69 ×10<sup>6</sup> (3.80 ×10<sup>6</sup> Bornean-exclusive) and 8.96 ×10<sup>6</sup> (5.19 ×10<sup>6</sup> Sumatran-exclusive) SNPs, respectively (<a class="paper__ref-link" href="#Fig5">Fig. 5</a>). Observing 36% more SNPs among Sumatran individuals strongly supports a larger <i>N</i><sub>e</sub>. In addition, independent analysis of 85 polymorphic retroelement loci among 37 individuals (19 Sumatran, 18 Bornean) also showed more complex Sumatran population structure (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 19). Using Watterson"s approach<sup><a class="paper__ref-link" href="#ref-CR24" id="ref-link-section-d34422e2744">24</a></sup>, we estimated nucleotide diversity from the SNP data as <i>θ</i><sub>W</sub> = 1.21 and <i>θ</i><sub>W</sub> = 1.62 per kb for the Bornean and Sumatran species, respectively, and <i>θ</i><sub>W</sub> = 1.89 per kb for the orang-utan species combined, roughly twice the diversity of modern humans<sup><a class="paper__ref-link" href="#ref-CR25">25</a></sup>.
          </p>
          <figure class="paper__figure">
            <figcaption id="Fig5">
              Figure 5: Orang-utan population genetics and demographics.
            </figcaption>
            <img src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_Article_BFnature09687_Fig5_HTML.jpg">
            <p>
              <b>a</b>, Site-frequency spectra for 13.2 × 10<sup>6</sup> Bornean (red) and Sumatran (blue) SNPs are shown based on the ascertainment of 10 chromosomes per species; note the enrichment of low-frequency SNPs among Sumatran individuals. <b>b</b>, The majority of SNPs were restricted to their respective island populations as the "heat" of the two-dimensional site-frequency spectra, representing high allele counts, lay along the axes. <b>c</b>, Our demographic model estimated that the ancestral orang-utan population (<i>N</i><sub>e</sub> = 17,900) split approximately 400,000 years ago, followed by exponential expansion of Sumatran <i>N</i><sub>e</sub> and a decline of Bornean <i>N</i><sub>e</sub>, culminating in higher diversity among modern Sumatran orang-utans despite a lower census population size. The model also supported low-level gene flow (<1 individual per generation), indicated by arrows.
            </p>
            <p>
              <a class="paper__ext-link" href="#MOESM278">PowerPoint slide</a>
            </p>
          </figure>
          <p>
            The modal category of SNPs were singletons, with 2.0 ×10<sup>6</sup> and 3.7 ×10<sup>6</sup> SNPs observed as single heterozygous sites in a Bornean or Sumatran individual, consistent with the expectation that most genetic variation for an outcrossing population ought to be rare due to mutation-drift equilibrium. We observed little correlation between Bornean and Sumatran SNPs in the allele frequency spectra (that is, the "heat" of the map is not along the diagonal as expected for populations with similar allele frequencies, but rather along the edges) (<a class="paper__ref-link" href="#Fig5">Fig. 5b</a>). This was further supported by principal component analysis, in which PC1 corresponded to the Bornean/Sumatran population label and explained 36% of the variance (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 20).</p><p>On the basis of these data, our demographic model consisted of a two-population model with divergence and potential migration, growth and difference in population size (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 21). Among several models tested, we found very strong statistical support (10<sup>5</sup> log-likelihood units) for the most complex model, which included a split with growth and subsequent low-level migration. We estimated a relative <i>N</i><sub>e</sub> of 210% for Sumatran orang-utans relative to the ancestral and 49% for Bornean orang-utans, noting a fourfold difference for the derived populations (<a class="paper__ref-link" href="#Fig5">Fig. 5c</a>). Assuming a mutation rate of 2.0 × 10<sup>−8</sup> and 20 years per generation, we estimated an ancestral <i>N</i><sub>e</sub> of 17,900 and a split time of 400,000 years ago.</p><p>Parallel to the SNP-based effort, we employed a coalescent hidden Markov model (coal-HMM) approach to estimate speciation time, recombination rate and ancestral <i>N</i><sub>e</sub> from the alignment of 20-fold coverage of a Bornean individual to the Sumatran reference (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 17). This method also supported a relatively recent Bornean/Sumatran speciation time (334 ± 145 kyr ago), and estimated a recombination rate of 0.95 ± 0.72 cM Mb<sup>−1</sup>. We independently estimated the ancestral <i>N</i><sub>e</sub> of the autosomes (26,800 ± 6,700) and the X chromosome (20,400 ± 7,400), which was consistent with the theoretical 3/4 effective population size of X chromosomes compared to autosomes. The Bornean and Sumatran X chromosome thus diverged as expected, in contrast to the human–chimpanzee speciation process<sup><a class="paper__ref-link" href="#ref-CR27">27</a></sup>.</p><p>The orang-utan story is thus a tale of two islands with distinct evolutionary histories. Our high-resolution population studies explored the counter-intuitive nature of orang-utan diversity—greater variation among Sumatran orang-utans than their Bornean counterparts despite a smaller population size (approximately sevenfold lower by recent estimates). Further dissection of the orang-utan speciation process will require a broader survey, incorporating representatives from additional orang-utan subpopulations.</p><p>Finally, even though we found deep diversity in both Bornean and Sumatran populations, it is not clear whether this diversity will be maintained with continued habitat loss and population fragmentation. Evidence from other species suggests fragmentation is not the death knell of diversity<sup><a class="paper__ref-link" href="#ref-CR28">28</a></sup>, but their slow reproduction rate and arboreal lifestyle may leave orang-utan species especially vulnerable to rapid dramatic environmental change. It is our hope that the genome assembly and population variation data presented here provide a valuable resource to the community to aid the preservation of these precious species.
          </p>
          <h2>Methods Summary</h2>
          <p>
            Whole-genome sequencing was performed as described previously<sup><a class="paper__ref-link" href="#ref-CR12">12</a>,<a class="paper__ref-link" href="#ref-CR13">13</a>,<a class="paper__ref-link" href="#ref-CR14">14</a></sup>. The genome assembly was constructed with a custom computational pipeline (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 1). Assembly source DNA was derived from a single Sumatran female (Susie; Studbook no. 1044; ISIS no. 71), courtesy of the Gladys Porter Zoo, Brownsville, Texas. Short fragment sequencing libraries for population studies (<a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> section 4) were constructed in accordance with standard Illumina protocols and sequenced on the Illumina GAIIx platform. The resulting data were processed with Illumina base-calling software and analysed using custom computational pipelines. See <a class="paper__ref-link" href="#MOESM279">Supplementary Information</a> for additional details.
          </p>
          <h2>Accession codes</h2>
          <p>DDBJ/GenBank/EMBL</p>
          <ul class="paper__list">
            <li class="paper__list-item">
              <a href="http://getentry.ddbj.nig.ac.jp/getentry?database=ddbj&amp;accession_number=ABGA00000000">ABGA00000000</a>
            </li>
            <li class="paper__list-item">
              <a href="http://getentry.ddbj.nig.ac.jp/getentry?database=ddbj&amp;accession_number=ABGA00000000.1">ABGA00000000.1</a>
            </li>
          </ul>
          <h3>Data deposits</h3>
          <p>
            The <i>P. abelii</i> whole-genome shotgun project has been deposited in DDBJ/EMBL/GenBank under the project accession <a href="http://getentry.ddbj.nig.ac.jp/getentry?database=ddbj&amp;accession_number=ABGA00000000">ABGA00000000</a>. The version described in this Letter is <a href="http://getentry.ddbj.nig.ac.jp/getentry?database=ddbj&amp;accession_number=ABGA00000000.1">ABGA00000000.1</a>. Assembly-based SNPs and SNPs derived from short read sequence data have been deposited in dbSNP. All short read data have been deposited into the Short Read Archive (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) under accessions listed in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nature09687#MOESM279">Supplementary Information</a>.
          </p>
        </section>
        <section class="paper__refs">    
          <h2>References</h2>
          <ol class="paper__refs-list">
            <li class="paper__refs-list-item">
              <p id="ref-CR1">Pontzer, H., Raichlen, D. A., Shumaker, R. W., Ocobock, C. &amp; Wich, S. A. Metabolic adaptation for low energy throughput in orangutans. <i>Proc. Natl Acad. Sci. USA</i> <b>107</b>, 14048–14052 (2010)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR2">van Noordwijk, M. A. &amp; van Schaik, C. P. Development of ecological competence in Sumatran orangutans. <i>Am. J. Phys. Anthropol.</i> <b>127</b>, 79–94 (2005)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR3">van Schaik, C. P. et al. Orangutan cultures and the evolution of material culture. <i>Science</i> <b>299</b>, 102–105 (2003)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR4">>Singleton, I. et al. Orangutan Population and Habitat Viability Assessment: Final Report (IUCN/SSC Conservation Breeding Specialist Group, Apple Valley, 2004)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR5">Meijaard, E. &amp; Wich, S. Putting orang-utan population trends into perspective. <i>Curr. Biol.</i> <b>17</b>, R540 (2007)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR6">Stanyon, R. et al. Primate chromosome evolution: ancestral karyotypes, marker order and neocentromeres. <i>Chromosome Res.</i> <b>16</b>, 17–39 (2008)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR7">Yi, S., Ellsworth, D. L. &amp; Li, W. H. Slow molecular clocks in Old World monkeys, apes, and humans. <i>Mol. Biol. Evol.</i> <b>19</b>, 2191–2198 (2002))</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR8">Hahn, M. W., Demuth, J. P. &amp; Han, S. G. Accelerated rate of gene gain and loss in primates. <i>Genetics</i> <b>177</b>, 1941–1949 (2007)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR9">Marques-Bonet, T. et al. A burst of segmental duplications in the genome of the African great ape ancestor. <i>Nature</i> <b>457</b>, 877–881 (2009)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR10">Seuanez, H., Fletcher, J., Evans, H. J. &amp; Martin, D. E. A chromosome rearrangement in orangutan studied with Q-, C-, and G-banding techniques. <i>Cytogenet. Cell Genet.</i> <b>17</b>, 26–34 (1976)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR11">Wade, C. M. et al. Genome sequence, comparative analysis, and population genetics of the domestic horse. <i>Science</i> <b>326</b>, 865–867 (2009)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR12">The Chimpanzee Sequencing and Analysis Consortium. Initial sequence of the chimpanzee genome and comparison with the human genome. <i>Nature</i> <b>437</b>, 69–87 (2005)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR13">International Human Genome Sequencing Consortium Initial sequencing and analysis of the human genome. <i>Nature</i> <b>409</b>, 860–921 (2001)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR14">Gibbs, R. A. et al. Evolutionary and biomedical insights from the rhesus macaque genome. <i>Science</i> <b>316</b>, 222–234 (2007)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR15">Esnault, C., Maestre, J. &amp; Heidmann, T. Human LINE retrotransposons generate processed pseudogenes. <i>Nature Genet.</i> <b>24</b>, 363–367 (2000)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR15">Esnault, C., Maestre, J. &amp; Heidmann, T. Human LINE retrotransposons generate processed pseudogenes. <i>Nature Genet.</i> <b>24</b>, 363–367 (2000)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR16">Liu, G. et al. Analysis of primate genomic variation reveals a repeat-driven expansion of the human genome. <i>Genome Res.</i> <b>13</b>, 358–368 (2003)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR17">Lee, J. et al. Different evolutionary fates of recently integrated human and chimpanzee LINE-1 retrotransposons. <i>Gene</i> <b>390</b>, 18–27 (2007)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR18">Kulpa, D. A. &amp; Moran, J. V. Cis-preferential LINE-1 reverse transcriptase activity in ribonucleoprotein particles. <i>Nature Struct. Mol. Biol.</i> <b>13</b>, 655–660 (2006)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR19">Bogerd, H. P. et al. Cellular inhibitors of long interspersed element 1 and Alu retrotransposition. <i>Proc. Natl Acad. Sci. USA</i> <b>103</b>, 8780–8785 (2006)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR20">Cordaux, R. &amp; Batzer, M. A. The impact of retrotransposons on human genome evolution. <i>Nature Rev. Genet.</i> <b>10</b>, 691–703 (2009)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR21">Kosiol, C. et al. Patterns of positive selection in six Mammalian genomes. <i>PLoS Genet.</i> <b>4</b>, e1000144 (2008)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR22">Ashburner, M. et al. Gene ontology: tool for the unification of biology. <i>Nature Genet.</i> <b>25</b>, 25–29 (2000)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR23">Makino, C. L. et al. Recoverin regulates light-dependent phosphodiesterase activity in retinal rods. <i>J. Gen. Physiol.</i> <b>123</b>, 729–741 (2004)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR24">Watterson, G. A. On the number of segregating sites in genetical models without recombination. <i>Theor. Popul. Biol.</i> <b>7</b>, 256–276 (1975)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR25">Li, W. H. &amp; Sadler, L. A. Low nucleotide diversity in man. <i>Genetics</i> <b>129</b>, 513–523 (1991)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR26">Hobolth, A., Christensen, O. F., Mailund, T. &amp; Schierup, M. H. Genomic relationships and speciation times of human, chimpanzee, and gorilla inferred from a coalescent hidden Markov model. <i>PLoS Genet.</i> <b>3</b>, e7 (2007)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR27">Patterson, N., Richter, D. J., Gnerre, S., Lander, E. S. &amp; Reich, D. Genetic evidence for complex speciation of humans and chimpanzees. <i>Nature</i> <b>441</b>, 1103–1108 (2006)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR28">Alcaide, M. et al. Population fragmentation leads to isolation by distance but not genetic impoverishment in the philopatric Lesser Kestrel: a comparison with the widespread and sympatric Eurasian Kestrel. <i>Heredity</i> <b>102</b>, 190–198 (2009)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR29">Yu, N. et al. Low nucleotide diversity in chimpanzees and bonobos. <i>Genetics</i> <b>164</b>, 1511–1518 (2003)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR30">Chen, F. C. &amp; Li, W. H. Genomic divergences between humans and other hominoids and the effective population size of the common ancestor of humans and chimpanzees. <i>Am. J. Hum. Genet.</i> <b>68</b>, 444–456 (2001)</p>
            </li>
            <li class="paper__refs-list-item">
              <p id="ref-CR30">Chen, F. C. &amp; Li, W. H. Genomic divergences between humans and other hominoids and the effective population size of the common ancestor of humans and chimpanzees. <i>Am. J. Hum. Genet.</i> <b>68</b>, 444–456 (2001)</p>
            </li>
          </ol>
          <h2>Acknowledgements</h2>
          <p>
            The orang-utan genome project was funded by the National Human Genome Research Institute (NHGRI), including grants U54 HG003079 (R.K.W.) and U54 HG003273 (R.A.G), with further support from National Institutes of Health R01 GM59290 (M.A.B.), PO1 AG022064 (M.A.B.), HG002385 (E.E.E.) and HG002238 (W.M.), National Science Foundation DBI-0644111 (A.S. and B.B.), David and Lucile Packard Foundation (A.S., V.T. and T.V.), Cornell University Provost"s Fellowship (A.L.M.), UK Medical Research Council (C.P.P., G.L., S.M. and A.H.), Marie Curie Fellowship (T.M.-B.), Ministerio de Ciencia e Innovación-Spain (MCI-Spain) and Fundación M. Botín (V.Q., X.S.P., G.R.O. and C.L.-O.), MCI-Spain BFU2006-15413-C02-01 and BFU2009-13409-C02-02 (A.N.), Spanish National Institute for Bioinformatics (INAB) and Fundação para a Ciência e a Tecnologia (Portugal), SFRH/BPD/26384/2006 (R.F.) and SFRH/BD/15856/2005 (O.F.), PRIN and CEGBA (M.R., N.A. and G.D.V.), and the Commission of the European Communities IRG-224885 (T.V.), IRG-231025 (B.B.). We thank the Gladys Porter Zoo, and S. O"Brien and S. Pääbo for use of orang-utan samples. D.P.L. thanks S. D. McGrath, A. Wollam, R. M. Abbott and L. Chemnick for technical assistance. We acknowledge all the important work that could not be cited owing to space limitations. Resources for exploring the orang-utan genome are available at UCSC (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>), Ensembl (<a href="http://www.ensembl.org">http://www.ensembl.org</a>), NCBI (<a href="http://ncbi.nlm.nih.gov">http://ncbi.nlm.nih.gov</a>) and The Genome Center at Washington University (<a href="http://genome.wustl.edu/genomes/view/pongo_abelii/">http://genome.wustl.edu/genomes/view/pongo_abelii/</a>).
          </p>
          <h2>Ethics declarations</h2>
          <p>The authors declare no competing financial interests.</p>
        </section>
        <footer class="paper__footer">
          <h2 id="MOESM279">Supplementary information</h2>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM279_ESM.pdf">Supplementary Information</a>
          <p class="paper__footer-link-info">This file contains Supplementary Information, Sections 1-22 (see Table of Contents), which include Supplementary Figures with legends, Supplementary Tables, Supplementary Methods and additional references. (PDF 11307 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM280_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This spreadsheet provides a general description of each cDNA data set generated for this project, as well as an internal experiment ID, flow cell ID, submission date and a Short Read Archive accession number. (XLS 24 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM281_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This spreadsheet provides a general description of each short read genomic DNA data set generated for this project, including species (Bornean/Sumatran), individual identification number, sex, flow cell ID, a description of the data type (read length and whether paired end or fragment data), and a Short Read Archive accession number for each lane of data. (XLS 31 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM282_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This file contains data with regard to variation in mean dS along the length of human chromosome 1, referenced in Supplemental Information section S7 (XLS 20 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM283_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This file contains data with regard to variation in mean dS along the length of orangutan chromosome 1, referenced in Supplemental Information section S7. (XLS 21 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM284_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This file contains data with regard to variation in dS in relation to genomic position and distance from structural elements of the genome (centromeres and telomeres) and rearranged regions of the genome, referenced in Supplemental information section S7. (XLS 161 kb)</p>
          <a class="paper__footer-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM285_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This file contains data with regard to variation in dS along the orangutan branch and hominid branch on a chromosome-by-chromosome basis, referenced in Supplemental Information section S7. (XLS 93 kb)</p>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM286_ESM.xls">Supplementary Data</a>
          <p class="paper__footer-link-info">This file contains data with regard to the effect of GC content on evolutionary rate variation in telomeric regions, in association with Supplemental Information section S7. (XLS 50 kb)</p>
          <h2>PowerPoint slides</h2>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM274_ESM.ppt">PowerPoint slide for Fig. 1</a>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM275_ESM.ppt">PowerPoint slide for Fig. 2</a>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM276_ESM.ppt">PowerPoint slide for Fig. 3</a>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM277_ESM.ppt">PowerPoint slide for Fig. 4</a>
          <a class="paper__footer-link-info" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature09687/MediaObjects/41586_2011_BFnature09687_MOESM278_ESM.ppt">PowerPoint slide for Fig. 5</a>
          <h2>Rights and permissions</h2>
          <p>This article is distributed under the terms of the Creative Commons Attribution-Non-Commercial-Share Alike licence (<a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" rel="license" itemprop="license">http://creativecommons.org/licenses/by-nc-sa/3.0/</a>), which permits distribution, and reproduction in any medium, provided the original author and source are credited. This licence does not permit commercial exploitation, and derivative works must be licensed under the same or similar licence.</p>
        </footer>
      </article>
    </main>
    <script>
    $(document).ready(function() {
      setupControls();
    });  
    </script>
  </body>
</html>